src/util/TchapUtils.ts

```diff
@@ -35,7 +35,7 @@ export default class TchapUtils {
         // Only show the federate switch to defense users : it's difficult to understand, so we avoid
         // displaying it unless it's really necessary.
         if (baseDomain === 'agent.intradef.tchap.gouv.fr') {
-            return { showRoomFederationOption: true, roomFederationDefault: false };
+            return { showRoomFederationOption: true, roomFederationDefault: true };
```

Comment:
@aulamber In case of "forum" default is false, in case private room (external or not, it's always federation default)

Reply:
Maybe you can change the isFederation :      [   this.setState({ tchapRoomType: tchapRoomType });](https://github.com/tchapgouv/tchap-web-v4/blob/0b25f2ece91ef1c417dc8ccaba5e22bdbf707acd/src/components/views/dialogs/TchapCreateRoomDialog.tsx#L82) if the room type isn't public room, isFederated is set to true.

It will be better to enforce the federation parameter at moment of the room is creation (I haven't found were it's the code of that).